renderdoc/driver/vulkan/wrappers/vk_device_funcs.cpp

```diff
@@ -1431,6 +1431,12 @@ bool WrappedVulkan::Serialise_vkCreateDevice(SerialiserType &ser, VkPhysicalDevi
          !strcmp(createInfo.ppEnabledExtensionNames[i], VK_EXT_DISPLAY_CONTROL_EXTENSION_NAME))
         continue;
 
+#if ENABLED(RDOC_NEW_IMAGE_STATE)
+      if(!strcmp(createInfo.ppEnabledExtensionNames[i],
+                 VK_KHR_SEPARATE_DEPTH_STENCIL_LAYOUTS_EXTENSION_NAME))
+        m_SeparateDepthStencilLayouts = true;
```

Comment:
I think this is the only comment I have not had time to address yet. I will switch this over to use `InstanceDeviceInfo` and look into the whitelisting next week.

Reply:
I've been looking into proper extension support (vs just supporting separate depth/stencil layouts in the image state tracking). I think I mostly have it working, and am currently working on a sample app that actually uses this extension so that I can verify this.

I think it might make sense to start a separate PR to actually add the extension support, along with the sample/test of the extension. I can revert this `m_SeparateDepthStencilLayouts` stuff in this PR.